packages/buckets/src/buckets.spec.ts

```diff
@@ -26,15 +26,21 @@ describe('Buckets...', () => {
 
   before(async () => {
     const user = await signUp(ctx, addrGatewayUrl, sessionSecret)
-    const id = ThreadID.fromRandom()
-    const db = new Client(ctx.withSession(user.user?.session).withThreadName('buckets'))
-    await db.newDB(id)
+    ctx.withSession(user.user?.session)
   })
 
-  it('should init a new bucket', async () => {
+  it('should open a bucket by name without thread info', async () => {
+    const root = await client.open('initbuck')
```

Comment:
open method can just take the name of the bucket. it will try to use the default 'buckets' thread if no thread name or thread id is provided. otherwise it will create it